packages/vgrammar/src/view/animate.ts

```diff
@@ -71,7 +85,22 @@ export class ViewAnimate implements IBaseAnimate {
     return this;
   }
 
+  private _onAnimationStart = (event: AnimationEvent) => {
+    if (this._animatingMarks.length === 0) {
```

Comment:
a mark may have two or more animation of different `animationState`
so `this._animatingMarks` should save the `mark` and `animationState`

Reply:
Fixed.